CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+cmake_minimum_required(VERSION 3.14 FATAL_ERROR)
+
+find_package(Git REQUIRED QUIET)
+
+project(CPM LANGUAGES NONE)
+
+if(GIT_EXECUTABLE)
```

Comment:
**NOTE**: this is not be needed and not reached with `REQUIRED` set

Reply:
true 👍🏻